src/box/space.h

```diff
@@ -38,6 +38,7 @@
 #include "index.h"
 #include "error.h"
 #include "diag.h"
+#include "tt_static.h"
```

Comment:
Are you sure this is needed? I dropped it and compilation worked.

Reply:
Oops, that's a leftover from some intermediate patch version. Removed, sorry.